From 88b0ad2bbdaac86dba4c9994fa734f1a649c6a1e Mon Sep 17 00:00:00 2001
From: mbegumx <mogalx.begum@intel.com>
Date: Thu, 29 Oct 2020 14:22:25 +0530
Subject: [PATCH] Updating the IMapper to 2.1

Removing IMapper version 2.0 inorder to fix the VTS failures in
VtsHalGraphicsMapperV2_1Target module because of two shared objects
getting created for IMapper in vendor/lib/hw/

Change-Id: I2883a4e7848add6d245ead915e982d7fd3cf43b3
Tracked-On: OAM-94310
Signed-off-by: mbegumx <mogalx.begum@intel.com>

diff --git a/groups/device-specific/caas/product.mk b/groups/device-specific/caas/product.mk
index ffe601c..c6f9ac3 100755
--- a/groups/device-specific/caas/product.mk
+++ b/groups/device-specific/caas/product.mk
@@ -20,7 +20,6 @@ PRODUCT_PACKAGES += android.hardware.keymaster@3.0-impl \
                     camera.device@1.0-impl \
                     android.hardware.camera.provider@2.4-impl \
                     android.hardware.camera.provider@2.4-service \
-                    android.hardware.graphics.mapper@2.0-impl \
                     android.hardware.graphics.mapper@2.0-impl-2.1 \
                     android.hardware.graphics.allocator@2.0-impl \
                     android.hardware.graphics.allocator@2.0-service \
2.29.0

